modules/react/button/lib/SecondaryButton.tsx

```diff
@@ -20,96 +20,104 @@ export interface SecondaryButtonProps extends ButtonProps {
-    [buttonVars.disabled.border]: base.blackPepper400,
-    [buttonVars.disabled.label]: base.blackPepper400,
-    [buttonVars.disabled.opacity]: '0.4',
-    [systemIconStencil.vars.color]: base.blackPepper400,
-  },
-});
-
-export const secondaryButtonModifiers = createModifiers({
-  variant: {
-    inverse: createStyles({
-      // Default Styles
-      [buttonVars.default.background]: 'transparent',
-      [buttonVars.default.border]: base.frenchVanilla100,
-      [buttonVars.default.label]: base.frenchVanilla100,
-      [systemIconStencil.vars.color]: base.frenchVanilla100,
-      // Focus Styles
-      '&:focus-visible, &.focus': {
-        [buttonVars.focus.background]: base.frenchVanilla100,
-        [buttonVars.focus.border]: base.frenchVanilla100,
-        [buttonVars.focus.label]: base.blackPepper400,
-        [buttonVars.focus.boxShadowInner]: base.blackPepper400,
-        [buttonVars.focus.boxShadowOuter]: base.frenchVanilla100,
-        [systemIconStencil.vars.color]: base.blackPepper400,
-      },
-      // Hover Styles
-      '&:hover, &.hover': {
-        [buttonVars.hover.background]: base.soap300,
-        [buttonVars.hover.border]: base.soap300,
-        [buttonVars.hover.label]: base.blackPepper500,
-        [systemIconStencil.vars.color]: base.blackPepper500,
-      },
-      // Active Styles
-      '&:active, &.active': {
-        [buttonVars.active.background]: base.soap400,
-        [buttonVars.active.border]: base.soap400,
-        [buttonVars.active.label]: base.blackPepper500,
-        [systemIconStencil.vars.color]: base.blackPepper500,
-      },
-      // Disabled Styles
-      '&:disabled, &.disabled': {
-        [buttonVars.disabled.background]: 'transparent',
-        [buttonVars.disabled.border]: base.frenchVanilla100,
-        [buttonVars.disabled.label]: base.frenchVanilla100,
-        [systemIconStencil.vars.color]: base.frenchVanilla100,
+  modifiers: {
+    variant: {
+      inverse: {
+        // Default Styles
+        [baseButtonStencil.vars.background]: 'transparent',
+        [baseButtonStencil.vars.border]: system.color.border.inverse,
+        [baseButtonStencil.vars.label]: system.color.fg.inverse,
+        [systemIconStencil.vars.color]: system.color.fg.inverse,
+        // Focus Styles
+        '&:focus-visible, &.focus': {
+          [baseButtonStencil.vars.background]: system.color.bg.default,
+          [baseButtonStencil.vars.border]: system.color.border.inverse,
+          [baseButtonStencil.vars.label]: system.color.fg.strong,
+          [baseButtonStencil.vars.boxShadowInner]: system.color.fg.strong,
+          [baseButtonStencil.vars.boxShadowOuter]: system.color.fg.inverse,
+          [systemIconStencil.vars.color]: system.color.fg.strong,
+        },
+        // Hover Styles
+        '&:hover, &.hover': {
+          [baseButtonStencil.vars.background]: system.color.bg.alt.default,
+          [baseButtonStencil.vars.border]: base.soap300,
```

Comment:
isn't this system color boder token?

Reply:
This was missing. 

I need to update this and see if this was included in the latest for tokens.

Reply:
Looks as if this is still missing